media_driver/linux/common/codec/ddi/media_ddi_decode_vp8.cpp

```diff
@@ -83,7 +83,7 @@ VAStatus DdiDecodeVP8::ParseSliceParams(
     num_token_partitions                = slcParam->num_of_partitions - 1;
     picParams->CodedCoeffTokenPartition = (num_token_partitions != 8) ? (num_token_partitions >> 1) : 3;
     //macroblock_offset is in unit of bit.
-    picParams->uiFirstMbByteOffset = slcParam->slice_data_offset + slcParam->macroblock_offset >> 3;
+    picParams->uiFirstMbByteOffset = slcParam->slice_data_offset + (slcParam->macroblock_offset + 7) >> 3;
```

Comment:
This looks wrong - slice_data_offset should be in bytes while macroblock_offset is in bits, while this is treating both as bits (with misleading bracketing).

Also, what is the rounding intended to do?  If the macroblock_offset isn't on a byte boundary then this will point the offset to the byte after the first one, which doesn't sound right.

Reply:
@fhvwy thanks, will add () to ensure the priority of operators, about the rounding, it is because HW requirement. if the macroblock offset is not on the edge of  a byte, HW need to add 1 to shift the position.